dkron/config.go

```diff
@@ -162,6 +164,9 @@ type Config struct {
 
 	// UI enable the web UI on this node. The node must be server.
 	UI bool
+
+	// UsageDisable disable sending anonymous usage stats
+	UsageDisable bool `mapstructure:"usage-disable"`
```

Comment:
I agree with making this abundantly clear to the user, but those are completely anon stats, I really need to have a good grasp of what's the real adoption of the tool.

Reply:
I'm not sure that you can call them completely anonymous. There isn't any directly identifiable information in the payload, however there are 2 items that seem indirectly identifiable.  

First, the HTTP request made to your stats server will contain the source IP address. While you can claim that this information is not recorded, there can be no proof of that. Secondly, the cluster ID is a hash of the config. This may be hard to reverse, but the structure is well known (the config struct) and unfortunately it also contains sensitive information (EncryptKey). This also means that every time you change the config of the cluster that this ID will change. 

Just adding some context for thought on this.

Reply:
I'm glad that this PR is raising some attention, this is a good indicator to start with, and I appreciate your feedback. Said that, yes, the IP address and the location will also be used to obtain stats, I will not claim they won't be recorded, but it's the same as when you are browsing any site, I know it's not completely equivalent but it's pretty common in servers too. As an example of what I want to do, here you have the stats of the KrakenD guys https://twitter.com/devopsfaith/status/1245621435839598593

What do you think will be a good way to communicate this telemetry? |I'm thinking on adding a big note in the first sections of the documentation.

Reply:
I'm thinking at the least it should be mentioned in the documentation on the website as well as in the ReadMe accompanying a dkron package.
How desirable would it be have the UI mention it at first use?